elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -203,21 +192,21 @@ class PfeContentSet extends PFElement {
 
   _toggleVisible() {
     if (this.isTab) {
-      if (this.tab) this.tab.removeAttribute("hidden");
+      if (this.tabs) this.tabs.removeAttribute("hidden");
       if (this.accordion) this.accordion.setAttribute("hidden", "");
     } else {
       if (this.accordion) this.accordion.removeAttribute("hidden");
-      if (this.tab) this.tab.setAttribute("hidden", "");
+      if (this.tabs) this.tabs.setAttribute("hidden", "");
     }
   }
 
   _removeNodes(list) {
-    let host = this.isTab ? this.tab : this.accordion;
-
     list.forEach(item => this._removeNode(item));
 
     // Check if the container is empty
-    if (!host.hasChildNodes()) host.setAttribute("hidden", "");
+    [this.tabs, this.accordion].forEach(host => {
+      if (!host.hasChildNodes()) host.setAttribute("hidden", "");
+    });
```

Comment:
Just account for both tabs and accordion to prevent edge cases.  Very clever.  It's going to run asynchronously which is totally fine. I just like to mentally make note of it for my own sake.

Reply:
I like the idea of parallelizing in the future though!